scripts/asm-installer/install_asm

```diff
@@ -84,6 +84,12 @@ ONLY_VALIDATE="${ONLY_VALIDATE:=0}"
 VERBOSE="${VERBOSE:=0}"
 PRINT_HELP=0
 
+# Environment variables used by Hub IDNS
+ENABLE_HUB_IDNS="${ENABLE_HUB_IDNS:=0}"
+ENVIRON_PROJECT_ID="${ENVIRON_PROJECT_ID:=}"
+HUB_MEMBERSHIP_ID="${HUB_MEMBERSHIP_ID:=}"
```

Comment:
The := means that users could set these environment variables before running the script and they would be honored in the script. Looking below, you unconditionally overwrite these variables, so I think the intention is to not let users set them. In that case, I would initialize these just to 0 and "".

Reply:
Hi @zerobfd, thanks a lot for reviewing! Yes I was to discuss about whether we want to give user option to set them. We can initialize these just to 0 and "" for 1.8 first release.